_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+
+This guide describes how to install Spinnaker in AWS or in an on-prem Kubernetes cluster with access to S3. It will create and use the following Amazon Web Services resources:
+
+- A Kubernetes cluster running on Amazon Web Services (AWS). EKS is a good way to get a Kubernetes cluster up on AWS - see the AWS documentation for this.
+- An Amazon S3 (Simple Storage Service) bucket. You can use an existing one or create a new one.
+- An NGINX Ingress controller in your AWS cluster.
+
+This document currently does not fully cover the following (see [Next Steps](#next-steps) for some links to achieve these)
+
+- TLS Encryption
+- Authentication/Authorization
+- Add K8s accounts to deploy to
+- Add cloud accounts to deploy to
+
+Note: This document is focused on Armory Spinnaker, but can be adapted to install Open Source Spinnaker by using a different Halyard container and a corresponding different Spinnaker version
+
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+# Assumptions / Prerequisites / Environments
+
+This document assumes the following:
+
+- You have a Kubernetes cluster up and running, with the following:
+
+  - You can access the Kubernetes API. If using EKS, either your user/role created the EKS cluster or your user/role has been added to the `aws-auth` configmap in the EKS cluster. See the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html) for more details.
+  - At least 2x worker nodes, each with at least 2 vCPUs and 4 GiB of memory. This is the bare minimum to install and run Spinnaker. If you're deploying more intermittent test workloads you will likely need more).
+
+- You have access to an S3 bucket or access to create an S3 bucket
+
+- You have access to an IAM role or user with access to the S3 bucket or can create an IAM role or user with access to the S3 bucket.
+
+This document is written with the following workflow in mind:
+
+- You have access to the EKS cluster, with `kubectl` configured and able to access the cluster, from your workstation
+- Halyard (the Spinnaker installer) will be run in the EKS cluster as a statefulset, with a single Kubernetes PersistentVolumeClaim
+
+Furthermore:
+
+- You will create AWS resources, such as service accounts, that will be permanently associated with your Spinnaker cluster
+
+# Installation Summary
+
+In order to install Spinnaker, this document covers the following things:
+
+- Creating a Kubernetes namespace to install everything in
+- Creating a Kubernetes service account with access to the namespace
+- Running Halyard (the Spinnaker installer) in the namespace
+- Creating an S3 bucket for Spinnaker to store persistent configuration in
+- Creating an IAM user that Spinnaker will use to access the S3 bucket (or alternately, granting access to the bucket via IAM roles)
+
+- Running the `hal` client interactively in the same Docker container, to:
```

Comment:
`Docker container` wait, are we doing Operator?  or did you mean Pod? 
I'm confused...

Reply:
changed!